### ISSUE TITLE
Wrong suggestions when addign a new repo

### ISSUE THREAD

**Issue by stof** [schmittjoh/php-code-analysis#31]
When adding a new repo on http://jmsyst.com/automated-code-reviews/repositories/new the autocompletion suggests all repos where I have write access. However, when selecting a repo where I don't have admin rights, it shows an error message and fails to enable the review.
The autocompletion should already filter them out instead of suggesting repos which don't work.

**Comment by schmittjoh** [comment]
I don't think that we can change this for now due to limitations of the GitHub API, but we can keep this open in case they extend their API.

**Comment by stof** [comment]
Well, AFAIK, travis is able to list only the repositories with admin access.

But anyway, the UI of the review seems to imply that the hook is optional (as you display whether it is activated or no) so it should not fail when it is unable to activate it.
